### DISCUSSION TITLE
Integration

### DISCUSSION THREAD

**Discussion by dilipubipark** [AxisCommunications/discussions#110] [VAPIX] [answered]
UbiPark is trying to integrate with the Axis System. Below is what we are trying to achieve:

1. A user will register with the UbiPark system providing their name, email, phone and their pass card number.
2. UbiPark will send the user information including the passcard number to AXIS system. The data will also contain a start and end time. This is the period for which the user would have access.
3. Axis system will store this information. (Whitelisting the user for access during the period)
4. When the user taps their pass card on an HID card reader that is linked to the Axis device.
5. The Axis system validates the card number and the validity of the card and allows access to the user by opening the gate.
6. The same system would be followed at exit
7. UbiPark system sends the activity data back to UbiPark which UbiPark will match to the registered user and use for reporting and other management purposes

- Axis device: AXIS 1001 Network Door Controller
- Card Reader: HID multiCLASS SE RP10
 
We have been working on integrating with the Axis System. We have been using the following link for documentation:

https://www.axis.com/vapix-library/subjects/t10067552/section/t10068774/display

 We have been given a parameter file that I have attached as well. Going through the documentation, it would be great if you could provide some guidance on the flow. Is the below flow correct?

• SetAccessPoint
• SetAccessController (Add AccessPoints)
• AccessProfile (Relation with AccessPoint)
• CredentialAccessProfile (Relation with AccessProfile)

   --------------------------------------------
And also need to confirm if the below values are to be used for AccessPoint Entity.

Brand = "AXIS"
ProdFullName = "AXIS A1001 Network Door Controller"
ProdNbr = "A1001"
ProdShortName = "AXIS A1001"
ProdType = "Network Door Controller"
ProdVariant = ""
WebURL = "http://www.axis.com/"

 Since this is our first time integrating with AXIS, we are a bit confused on the steps and direction.

**Comment by mattias-kindborg-at-work** [comment]
Hi @dilipubipark. Let me see if I can find the information you are requesting.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @dilipubipark! You seem to have gotten in contact with Axis employees helping you with your integration. Best of luck to you and your endeavours going forward!

**Reply by dilipubipark** [reply]
Thank you @mattias-kindborg-at-work ! Yes. They did reach out and have got things moving. Cheers.
